src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/CastHelpers.cs

```diff
@@ -214,6 +215,55 @@ private static CastResult TryGet(nuint source, nuint target)
                 if (interfaceCount != 0)
                 {
                     MethodTable** interfaceMap = mt->InterfaceMap;
+
```

Comment:
Rather than adding code here, could all of this just be replaced with a call to SpanHelpers.ContainsValueType?

Reply:
Well I didn't realize it can be spanified. That would be better if it doesn't hurt R2R etc.